### ISSUE TITLE
Question about docker and "elevation correction with SRTM.py"

### ISSUE THREAD

**Issue by tushev** [eneiluj/gpxpod#24]
Hello,

Is this still relevant: https://github.com/julien-nc/gpxpod/blob/main/docs/admin.md#install-srtmpy-on-ubuntudebian ?

or
> GpxPod does not need python librairies anymore since v2.0.0 !

this applies?

I'm running NextCloud in Docker, so if yes, I guess I have to modify my setup.

**Comment by julien-nc** [comment]
I had a look at the code, SRTM.py is still used when correcting elevations in the old UI. I think I'll get rid of this and replace it with the new Php-based implementation in the next release.

Out of curiosity: Are you using this feature to add elevation data in tracks that don't have any or are you using it to fix imprecise elevations?

If you use it to fix the elevation data, I would be curious to know how you feel about the result. For most of the tracks I tested it on (with SRTM.py and with my Php-based one) the results were quite poor, especially for tracks with high point density as the SRTM grid resolution is quite big (50m or 90m, I don't remember).

**Comment by tushev** [comment]
>Are you using this feature to add elevation data in tracks that don't have any or are you using it to fix imprecise elevations?

No, I just decided to read the docs thoroughly and noticed that feature that I've not implemented in my containers.
If I would be using it, just to add elevation to the old tracks with missing height...
